bundles/org.openhab.core/src/main/resources/OH-INF/config/addons.xml

```diff
@@ -17,6 +17,16 @@
 				expected. Enabling this option will include these entries in the list of available add-ons.</description>
 			<default>false</default>
 		</parameter>
+		<parameter name="suggestionFinderUpnp" type="boolean">
```

Comment:
Should we mark those as `advanced`, so that the normal user does not even have to bother seeing and understanding them?